colibris/views/api.py

```diff
@@ -9,6 +11,14 @@ class APIView(View):
     body_schema_class = None
     query_schema_class = None
 
+    async def _iter(self):
+        await self.prepare()
+
+        return await super()._iter()
+
+    async def prepare(self):
+        self.request.body_text = await self.request.text()
```

Comment:
@ccrisan, I'm not sure why, but this is the only way it works. The syntax that we discussed (`[]`) it does not seem to work.

Reply:
@lucifurtun what do you mean it does not seem to work? This is what `aiohttp` officially recommends: https://docs.aiohttp.org/en/stable/web_advanced.html#data-sharing-aka-no-singletons-please.